src/Oxid/Core/MakairaConnectOutput.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Makaira\OxidConnectEssential\Oxid\Core;
+
+use OxidEsales\Eshop\Core\Exception\ArticleInputException;
+use OxidEsales\Eshop\Core\Exception\NoArticleException;
+use OxidEsales\Eshop\Core\Registry;
+use OxidEsales\EshopCommunity\Core\Output;
+
+/**
+ * This file is part of a marmalade GmbH project
+ * It is not Open Source and may not be redistributed.
+ * For contact information please visit http://www.marmalade.de
+ * Version:    1.0
+ * Author:     Sunny <dt@marmalade.de>
+ * Author URI: http://www.marmalade.de
+ */
+class MakairaConnectOutput extends Output
```

Comment:
The class name is not correct for an OXID extension.
It must be `MakairaConnectOutput_parent`, otherwise it breaks the module chain. The parent class is dynamically created in memory by OXID (using [`class_alias`](https://www.php.net/manual/en/function.class-alias.php))

```suggestion
class MakairaConnectOutput extends MakairaConnectOutput_parent
```